plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java

```diff
@@ -57,22 +60,14 @@ public void testFromLaunchConfigurationCopiesArgumentsFromLaunchConfiguration()
       throws CoreException {
     ILaunchConfiguration launchConfiguration = mock(ILaunchConfiguration.class);
     Map<String, String> requiredArgumentValues =
-        ImmutableMap.<String, String>builder()
-            .put("Spam", "foo")
-            .put("Ham", "bar")
-            .put("Eggs", "baz")
-            .build();
-    when(
-        launchConfiguration.getAttribute(
-            PipelineConfigurationAttr.ALL_ARGUMENT_VALUES.toString(),
-            Collections.<String, String>emptyMap())).thenReturn(requiredArgumentValues);
+        ImmutableMap.of("Spam", "foo","Ham", "bar", "Eggs", "baz");
```

Comment:
space between foo and Ham though I think I find the original version a little easier to read.

Reply:
OK, I can make one pair per line.